spec.md

```diff
@@ -1422,8 +1437,21 @@ message Error {
       // terminated or deleted before retrying.
       NODE_DOES_NOT_EXIST = 6;
 
+      // Indicates that the specified `NodeID` is not allowed or
+      // understood by the Plugin. More human-readable information MAY
+      // be provided in the `error_description` field.
+      //
+      // Recovery behavior: Caller MUST fix the `NodeID` before
+      // retrying.
       INVALID_NODE_ID = 7;
+
       VOLUME_NOT_ATTACHED_TO_SPECIFIED_NODE = 8;
+
+      // Indicates that the `NodeID` is not set to match that returned
+      // by `GetNodeID`.
```

Comment:
This is not quite precise. There is another case where the CO just want to unpublish the volume no matter which node the volume is attached to. I think this error code is for those plugins that do not support this feature? What will be the logic of an CO when receiving this error code? What if the node ID has been lost? Thoughts? @saad-ali @jdef